src/client/editor/components/TransformComponent.js

```diff
@@ -21,12 +21,15 @@ export default class TransformComponent extends BaseComponent {
     };
   }
 
-  get props() {
-    return TransformComponent._propsFromObject(this._object);
+  constructor(node, object) {
+    super(node, object);
+    Object.defineProperty(this, "props", {
```

Comment:
What is this for?

Reply:
TransformComponent is special in that it doesn't store its own position, rotation and scale properties, but it reads it directly from the Object3D

Reply:
Maybe leave a comment there. Seems good though.